src/Symfony/Bridge/Twig/Extension/AssetExtension.php

```diff
@@ -20,7 +20,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  *
- * @final since Symfony 4.4
+ * @final
```

Comment:
Can't we move it to a real final class? `final class AssetExtension ...`

Reply:
Certainly. @nicolas-grekas?

Reply:
well, we should do it in master indeed.